### PR TITLE
update cloud jobs

### PR DESCRIPTION
**Pull request by huali9**
update cloud jobs to cover both techpreview and non-techpreview.
@sunzhaohua2 @miyadav @shellyyang1989 PTAL, thanks!

### CODE REVIEW

**Comment by miyadav** [comment]
@huali9  can we trigger any one of them now to check ?

**Comment by huali9** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-aws-ipi-disc-priv-amd-mixarch-f9-longduration-cloud

**Comment by huali9** [comment]
The result looks good. https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/50271/rehearse-50271-periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-aws-ipi-disc-priv-amd-mixarch-f9-longduration-cloud/1772865727343104000

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml [hunk 1]
```diff
@@ -184,7 +184,7 @@ tests:
     - ref: ipi-install-heterogeneous
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
-- as: aws-ipi-disc-priv-amd-mixarch-f9-longduration-cloud
+- as: aws-ipi-disc-priv-amd-mixarch-tp-f9-longduration-cloud
```
Comment (sunzhaohua2) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:187]: dont think we need to run every 9 days, can we update to 14? tp clusters to 28?

Comment (huali9) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:187]: It was changed to 9 by @liangxia before https://github.com/openshift/release/pull/49982/files, can we change it back?

Comment (huali9) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:187]: @liangxia We have four jobs were changed to f9, they were f28 before, not sure about the background, can we change it back? They are long duration jobs for cloud team, which may take 13 hours for each job.
aws-ipi-disc-priv-amd-mixarch-f9-longduration-cloud and azure-ipi-disc-fullypriv-amd-mixarch-f9-longduration-cloud for 4.15 and 4.16

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:187]: The higher frequency are for component readiness.
How about keep it as it's now for some time ? We will revisit it after we onboard component readiness.

Comment (huali9) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:187]: > How about keep it as it's now for some time ? We will revisit it after we onboard component readiness.

Thank you @liangxia I am ok, what do you think? @sunzhaohua2

Comment (sunzhaohua2) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:187]: OK for me @huali9

**Pull request merged**